OnlineShop.DAL/Repositories/UserRepository.cs

```diff
@@ -1,11 +1,25 @@
-﻿using MyOnlineShop.DAL.Interfaces;
+﻿using Microsoft.EntityFrameworkCore;
+using OnlineShop.DAL.Interfaces;
 
-namespace MyOnlineShop.DAL.Repositories
+namespace OnlineShop.DAL.Repositories
 {
     public class UserRepository : GenericRepository<User>, IUserRepository
     {
         public UserRepository(OnlineShopDbContext context) : base(context)
         {
         }
+
+        public async Task AddAsync(User user)
```

Comment:
you need to throw the token into the methods of the controller, then into the methods of the service and then into the methods of the repositories and into the repositories themselves